controllers/storagecluster/cephcluster.go

```diff
@@ -1308,8 +1308,8 @@ func isBluestore(store rookCephv1.OSDStore) bool {
 	return false
 }
 
-func (r *StorageClusterReconciler) getOsdCount(sc *ocsv1.StorageCluster) int {
-	storageClassDeviceSets := newStorageClassDeviceSets(sc, r.serverVersion)
+func getOsdCount(sc *ocsv1.StorageCluster, serverVersion *version.Info) int {
```

Comment:
@malayparida2000 FYI. I made this a function so that we can invoke it without `StorageClusterReconciler`.

Reply:
ack, The getOsdCount function looks good.